models/Exercise.js

```diff
@@ -38,6 +38,11 @@ ExerciseSchema.statics.getExo = function (data, callback) {
     Exercise.findOne(query, callback);
 };
 
+ExerciseSchema.statics.ByAuthor = function (author, callback) {
+    let query = { author: author };
```

Comment:
Es6 bro : {author: author} should be {author}

Reply:
Fixed